### DISCUSSION TITLE
Access report folder from IScenarioInitContext

### DISCUSSION THREAD

**Discussion by candritzky** [VIP-Logic/NBomber#556] [General]
Is there a way to get the full path of the report folder from within the Scenario Init and Clean handlers?
I'm thinking of some property on IScenarioInitContext.
The idea is that I want to store additional reports (captured performance metrics) into the report folder from within the Clean handler.

**Comment by AntyaDev** [comment]
Hi Chris,

The current version does not support it.
But it can be added :)
